scripts/pr2/execute_pr2.py

```diff
@@ -18,20 +18,24 @@
     parser.add_argument("-t", type=float, default=300, help="time out")
     parser.add_argument("--force", action="store_true", help="disable dry option")
     parser.add_argument("--roseus", action="store_true", help="command via roseus")
+    parser.add_argument("--terminate", action="store_true", help="use terminate flag")
```

Comment:
verb "use" in "use terminate flag" is bit ambiguous. something like `terminate using terminate flag` is better